UPGRADE-4.2.md

```diff
@@ -257,6 +257,7 @@ TwigBundle
 ----------
 
  * The `transchoice` tag and filter have been deprecated, use the `trans` ones instead with a `%count%` parameter.
+ * Deprecated support for legacy templates directories `src/Resources/views/` and `src/Resources/<BundleName>/views/`, use `templates/` and `templates/bundles/<BundleName>/` instead.
```

Comment:
You mean `app/Resources/views/` and `app/Resources/<BundleName>/views/`, don't you?

Reply:
I meant `src/` yes, it's the default Kernel directory since 4.0, no? at this point, shouldn't exist `app/`

Reply:
Not only, as `%kernel.root_dir%` is `src` using Flex.

Reply:
indeed, you are right